build.sbt

```diff
@@ -1536,7 +1536,14 @@ lazy val zioHttpServer: ProjectMatrix = (projectMatrix in file("server/zio-http-
   .settings(commonJvmSettings)
   .settings(
     name := "tapir-zio-http-server",
-    libraryDependencies ++= Seq("dev.zio" %% "zio-interop-cats" % Versions.zioInteropCats % Test, "dev.zio" %% "zio-http" % "3.0.0-RC4")
+    libraryDependencies ++= Seq(
+      "dev.zio" %% "zio-interop-cats" % Versions.zioInteropCats % Test,
+      "dev.zio" %% "zio-http" % "3.0.0-RC5",
+      // Workaround for https://github.com/zio/zio-http/issues/2726
+      "dev.zio" %% "zio-schema" % "1.0.1",
+      "dev.zio" %% "zio-schema-json" % "1.0.1",
+      "dev.zio" %% "zio-schema-protobuf" % "1.0.1"
```

Comment:
This is really unfortunate, but it's required at the moment. Do you prefer if we add a `// scala-steward:off` comment so that they don't get updated?